design/one-pager-stacks-security-isolation.md

```diff
@@ -95,11 +122,14 @@ Let's look at a specific example of the [GCP cluster stack](https://github.com/c
 First, the user will create an instance of a `ClusterStackInstall` that specifies which cluster stack they want to install.
 Then, when the Stack Manager installs the GCP stack, a few things will happen:
 
-* All CRDs related to GCP infrastructure and services will be installed to the cluster, such as `CloudSQLInstance`, `CloudSQLInstanceClass`, `GKECluster`, `GKEClusterClass`, `Network`, `Subnetwork`, etc.
-* A `ClusterRole` will be created at the cluster level that includes the RBAC rules for the [allowed core Kubernetes kinds](#allowed-resource-access), the stack's owned CRDs, and the CRDs that it depends on.
+* All CRDs related to GCP infrastructure and services will be installed to the cluster, such as `CloudSQLInstance`,
+ `CloudSQLInstanceClass`, `GKECluster`, `GKEClusterClass`, `Network`, `Subnetwork`, etc. All CRDs will be labeled
+ `crossplane.environment=true` to denote that they are from a cluster scoped install and for the environment.
+* A `ClusterRole` is created for for system that includes the RBAC rules for the [allowed core Kubernetes kinds](#allowed-resource-access), the stack's owned CRDs, and the CRDs that it depends on.
+* `ClusterRoles` for admin, edit, and view will be created in the cluster scope for the stacks owned CRDs.
 * A `ServiceAccount` is created in the namespace that the stack's controller will be running.  This will serve as the runtime identity of the controller.
-* A `ClusterRoleBinding` is created at the cluster level to bind the `ClusterRole` and the namespaced `ServiceAccount`
-* A `Deployment` is created in the same namespace as the stack and it is expected to launch and manage the pod that hosts the GCP controllers.
+* A `ClusterRoleBinding` is created at the cluster level to bind the system `ClusterRole` and the namespaced `ServiceAccount`
+* A `Deployment` is created in the crossplane-system namespace and it is expected to launch and manage the pod that hosts the GCP controllers.
```

Comment:
Is there a reason we need to specifically call out `crossplane-system` here? Currently that namespace is by convention the place we install Crossplane and its stacks, rather than something we hard-code or require anywhere.

Reply:
I've updated this to reflect that it will be installed as defined in the Spec of the ClusterStackInstall. I'll leave that up to an implementation detail as to which pattern is chosen.